glslc/src/main.cc

```diff
@@ -1,4 +1,5 @@
 // Copyright 2015 The Shaderc Authors. All rights reserved.
+// MODIFIED BY Robin Quint
```

Comment:
this isn't necessary, please remove.
By committing to this repo you become part of the collective of "Shaderc Authors".
If you like you can add yourself to the CONTRIBUTORS file in the root of the source tree.  But authorship is determined by the git history.